dev-ruby/certificate_authority/certificate_authority-1.0.0-r1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
There has been an upstream release with this commit and I've added that, so this commit is no longer needed.